src/udev/udevadm-info.c

```diff
@@ -469,12 +469,165 @@ static int stat_device(const char *name, bool export, const char *prefix) {
         return 0;
 }
 
-static int export_devices(sd_device_enumerator *e) {
-        sd_device *d;
+static int ensure_device_enumerator(sd_device_enumerator **e) {
```

Comment:
Nah, please squash the change into an earlier commit.

Reply:
I'd prefer to keep the main commit as small as possible and this doesn't fit anywhere else...

Reply:
OK. Not a big matter. Please keep the current form if you'd like.